pharus/component_interface.py

```diff
@@ -226,8 +264,46 @@ def __init__(self, *args, **kwargs):
 
     def dj_query_route(self):
         fetch_metadata = self.fetch_metadata
-        return (fetch_metadata["query"] & self.restriction).fetch1(
-            *fetch_metadata["fetch_args"]
+        record_header, table_records, total_count = _DJConnector._fetch_records(
+            query=fetch_metadata["query"] & self.restriction,
+            fetch_args=fetch_metadata["fetch_args"],
+            fetch_blobs=True,
+        )
+        return NumpyEncoder.dumps(table_records[0][0])
+        # return (fetch_metadata["query"] & self.restriction).fetch1(
+        #     *fetch_metadata["fetch_args"]
+        # )
+
+
+class SliderComponent(QueryComponent):
```

Comment:
Is this component any different from the `MetadataComponent`?

Reply:
Yes it does not need/have an attribute route, since metadata inherits from table it has an attribute route.